corehq/apps/users/migrations/0009_sqlinvitation_squashed_0012_alter_usersinvitation.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.2.11 on 2020-04-09 18:53
+
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('users', '0009_sqlinvitation'), ('users', '0010_populate_usersinvitation'), ('users', '0011_sqlinvitation_uuid'), ('users', '0012_alter_usersinvitation')]
+
+    dependencies = [
+        ('users', '0008_auto_20200129_1852'),
+    ]
+
+    operations = [
```

Comment:
This is missing the `RunPython` from the old 0010, so the new table isn't going to get populated.

Squashing 0011 and 0012 is fine, but squashing all four means the migration won't work as intended even if the `RunPython` is added, because invitations created while the deploy/migration is running may not be correctly migrated to sql.

This might be fine, because our environments have all had all of these migrations run, and third parties are unlikely to even use invitations, but I wanted to be sure you were aware.